spec/forms/school_search_form_spec.rb

```diff
@@ -51,7 +52,8 @@
     it "filters schools by name and partnership status" do
       school = schools[2]
       lead_provider = FactoryBot.create(:lead_provider)
-      Partnership.create!(school: school, lead_provider: lead_provider)
+      cohort = FactoryBot.create(:cohort, start_year: 2021)
```

Comment:
ditto

Reply:
Currently, the partnership model requires cohort_id , because the db is cleaned every time - no cohort exist. If I remove that - the test fails. Perhaps I've misunderstood the comment - more than happy to have a chat about it

Reply:
Done